eze/src/main/kotlin/org/camunda/community/eze/engine/InMemoryLogStorage.kt

```diff
@@ -37,11 +37,13 @@ class InMemoryLogStorage : LogStorage {
     override fun append(
         lowestPosition: Long,
         highestPosition: Long,
-        blockBuffer: ByteBuffer,
+        bufferWriter: BufferWriter,
         listener: LogStorage.AppendListener
     ) {
         try {
-            val entry = Entry(blockBuffer)
+            val copy: DirectBuffer = BufferUtil.createCopy(bufferWriter)
```

Comment:
@saig0 why do you think this was necessary now?

Reply:
I kept the original structure of the in-memory log storage. Since the API changed, I needed to extract the content of the `BufferWriter` to fill the entry.

But maybe, there is another way. I'm open to your ideas. :smile: